File: tests/fixtures/find-comment-drift-csharp/host/src/CSharpComment/CommentEvidence.cs
namespace CSharpComment;

public static class CommentEvidence
{
    public const string UnicodePrefix = "café 😀";

    // SiteConfig still lives at BillingParser.cs:42.
    /* SECTION 12 BILLING PARSERS */
    /// Parse the invoice state.
    public static string Normalize(string raw) => raw.Trim();
}

File: tests/fixtures/kotlin-comment-map/overlay/src/main/kotlin/cohort/comments/CommentEvidence.kt
package cohort.comments

import cohort.Invoice
import cohort.parseBilling

// SiteConfig still lives at BillingParser.kt:42.
// SECTION 12 BILLING PARSERS
// Parse the invoice state.
private fun normalize(raw: String): String = raw.trim()

internal fun internalLabel(invoice: Invoice): String = invoice.id

fun parseCommentEvidence(raw: String): Invoice = parseBilling(normalize(raw))

const val COMMENT_LOOKING_STRING = "// SiteConfig still lives at BillingParser.kt:42."

// Preserve this spelling because external fixture assertions depend on it.
fun stableCommentEvidence(): String = COMMENT_LOOKING_STRING
